### ISSUE TITLE
Generating Pdf's does not work

### ISSUE THREAD

**Issue by AlexProductions** [DavBfr/dart_pdf#1394]
**Describe the bug**
pdf 3.9.0 was the last version that worked.
Since upgrading to Flutter 3.10.1 (and Dart 3), that version stopped to work and i was forced to use 3.10.3.

currently, generating any pdf does fail at runtime.

**To Reproduce**
Code snippet to reproduce the behavior:
```dart
Scaffold(
  body: PdfPreview(build: (format) {
    pw.Document d = pw.Document();
     d.addPage(pw.Page(
        build: (context) {
          return pw.Text("Hey");
        },
     ));
   
    return d.save();
     },),
);
```

**Expected behavior**
Showing the pdf

**Screenshots**
(There is just a blank, grey screen at production and a red one showing the exception in debug mode)

**Flutter Doctor**

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.10.1, on macOS 13.4 22F66 darwin-arm64, locale de-DE)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.2)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.2)
[✓] Connected device (2 available)
[✓] Network resources

```

**Desktop (please complete the following information):**
 - [ ] iOS
 - [ ] Android
 - [✓] Browser
 - [ ] Windows
 - [ ] Linux

**Smartphone (please complete the following information):**
 - Browser: latest Google Chrome and Microsoft Edge

**Additional context**
```
TypeError: Cannot set properties of null (setting 'exports')
https://unpkg.com/pdfjs-dist@3.2.146/build/pdf.min.js 22:113  webpackUniversalModuleDefinition
https://unpkg.com/pdfjs-dist@3.2.146/build/pdf.min.js 22:294  <fn>
Error: NoSuchMethodError: '[]'
Dynamic call of null.
Receiver: null
Arguments: ["GlobalWorkerOptions"]
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 288:49      throw_
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 813:3   defaultNoSuchMethod
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 807:56  noSuchMethod
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 283:12  callNSM
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 294:25  _checkAndCall
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 431:10  callMethod
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 435:5   dsend
packages/printing/printing_web.dart 90:29                                         _initPlugin
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 45:50                <fn>
dart-sdk/lib/async/zone.dart 1661:54                                              runUnary
dart-sdk/lib/async/future_impl.dart 147:18                                        handleValue
dart-sdk/lib/async/future_impl.dart 784:44                                        handleValueCallback
dart-sdk/lib/async/future_impl.dart 813:13                                        _propagateToListeners
dart-sdk/lib/async/future_impl.dart 575:7                                         [_complete]
dart-sdk/lib/async/stream_pipe.dart 61:11                                         _cancelAndValue
dart-sdk/lib/async/stream.dart 1581:7                                             <fn>
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 367:37  _checkAndCall
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 372:39  dcall
dart-sdk/lib/html/dart2js/html_dart2js.dart 37263:58                              <fn>
Helvetica has no Unicode support see https://github.com/DavBfr/dart_pdf/wiki/Fonts-Management
```

**Comment by rodolfogoulart** [comment]
> Same here, in my case it works in debug mode, but it doesn't in release (only web)
> 
> ![image](https://user-images.githubusercontent.com/14177913/240630466-172664f6-cef0-4e33-8b4f-1f299f84b231.png)

For me to, in Windows, work on debug but not in release.. .

One thing i noted, is when insert actions don't work on release.. if you delete actions, work....

**Comment by DavBfr** [comment]
Should be fixed with printing 5.11

**Comment by AlexProductions** [comment]
Yes, the problem is solved now. Thank you :)
